Objects/unicodeobject.c

```diff
@@ -15063,9 +15175,74 @@ _PyUnicode_InitTypes(PyInterpreterState *interp)
     return _PyStatus_ERR("Can't initialize unicode types");
 }
 
-
 void
-_PyUnicode_InternInPlace(PyInterpreterState *interp, PyObject **p)
+_PyUnicode_InternStatic(PyInterpreterState *interp, PyObject **p)
+{
```

Comment:
Are there any situations where this would be meaningful outside runtime initialization?  IOW, could we expect that this would only be called during main interpreter init?  If so, a couple of things in this function maybe could be a little simpler.

Reply:
FTR: https://github.com/python/cpython/issues/113993#issuecomment-2178284917